src/CartBlock.C

```diff
@@ -20,6 +20,7 @@
 #include "codetypes.h"
 #include "CartBlock.h"
 #include "CartGrid.h"
+#include "cartUtils.h"
```

Comment:
It looks like you probably forgot to add the header to the git commit?

Reply:
Thanks for catching it! I have ammended the commit - d0b82f4